haas/ext/switches/_console.py

```diff
@@ -119,6 +119,13 @@ def revert_port(self, port):
         self.exit_if_prompt()
         self.console.expect(self.config_prompt)
 
+    def _should_save(self, switch_type):
+        """checks the config file to see if switch should save or not"""
+
+        switch_ext = 'haas.ext.switches.' + switch_type
+        return (not cfg.has_option(switch_ext, 'save') or
```

Comment:
Yeah, the python2 configparser api is not great. I don't feel strongly on this point.